src/compiler/clang.rs

```diff
@@ -176,6 +176,7 @@ counted_array!(pub static ARGS: [ArgInfo<gcc::ArgData>; _] = [
     take_arg!("-MF", PathBuf, CanBeSeparated, DepArgumentPath),
     take_arg!("-MQ", OsString, CanBeSeparated, DepTarget),
     take_arg!("-MT", OsString, CanBeSeparated, DepTarget),
+    take_arg!("-Xarch", OsString, ConcatenatedAndSeparated('_'), PassThrough),
```

Comment:
Is `PassThough` the right choice here? I only dealt with the mechanism of recognizing and processing the arg, but not the semantics of passing it and how that may affect the cache.